### ISSUE TITLE
Echeyakee Lair

### ISSUE THREAD

**Issue by Krevro** [dalaranwow/dalaran-wow#4882]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Upon reaching Echeyakee's lair your quest item to summon him doesn't
![wowscrnshot_092017_214646](https://user-images.githubusercontent.com/32154177/30679400-2ac45e50-9e4e-11e7-9c6f-a62457ad29a0.jpg)
![wowscrnshot_092017_214646](https://user-images.githubusercontent.com/32154177/30679408-3a207dde-9e4e-11e7-9319-77e2f59162ba.jpg)

 recognize the area as being his lair so you can't proceed with the quest. 
**Expected behaviour**: Tell us what should happen instead
Your quest item should summon Echeyakee when used if you are at his lair. 
**Steps to reproduce the problem**:

1. Accept quest
2. Go to Echeyakee's lair 
3. Try to use quest item 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Works 100% fine. You're standing in the wrong location. Go down to the bones. Close this report please.

**Comment by Crispybuilt** [comment]
Working as intended, close this up.

**Comment by TeridanDW** [comment]
Invalid, answer provided by @Chenocide.
